integration/feature/full-run-logs/src/FullRunLogsTests.scala

```diff
@@ -74,5 +55,41 @@ object FullRunLogsTests extends UtestIntegrationTestSuite {
       assert(millProfile.exists(_.obj("label").str == "show"))
       assert(millChromeProfile.exists(_.obj("name").str == "show"))
     }
+    test("failedTasksCounter") - integrationTest { tester =>
+      import tester._
+
+      // First ensure clean compilation works
+      val cleanBuild = eval(("compile"))
+      cleanBuild.isSuccess ==> true
+
+      // Modify the Java source to introduce a compilation error
+      val javaSource = os.Path(workspacePath, os.pwd)
+      val fooJava = javaSource / "src" / "foo" / "Foo.java"
+      val originalContent = os.read(fooJava)
```

Comment:
No need to check clean compile works, we have other tests for that. Can just start with broken code